coderd/database/models.go

```diff
@@ -3845,6 +3847,8 @@ type WorkspaceBuild struct {
 	DailyCost               int32               `db:"daily_cost" json:"daily_cost"`
 	MaxDeadline             time.Time           `db:"max_deadline" json:"max_deadline"`
 	TemplateVersionPresetID uuid.NullUUID       `db:"template_version_preset_id" json:"template_version_preset_id"`
+	HasAiTask               bool                `db:"has_ai_task" json:"has_ai_task"`
+	AiTasksSidebarAppID     uuid.NullUUID       `db:"ai_tasks_sidebar_app_id" json:"ai_tasks_sidebar_app_id"`
```

Comment:
Should probably not be a `uuid.NullUUID` since this field is required in terraform.
Ditto for the `Ai`->`AI` fix.

Reply:
Gotcha 👍 no need, you can leave it as is.